openwisp_firmware_upgrader/base/models.py

```diff
@@ -254,6 +254,12 @@ class AbstractDeviceFirmware(TimeStampedEditableModel):
     )
     image = models.ForeignKey(get_model_name('FirmwareImage'), on_delete=models.CASCADE)
     installed = models.BooleanField(default=False)
+    no_preserve_settings = models.BooleanField(default=False)
```

Comment:
I think we need a better way to display this to the user without including it as part of a FirmwareImage